packages/design-system-react/jest.config.js

```diff
@@ -15,7 +15,7 @@ module.exports = merge(baseConfig, {
   displayName,
 
   // Add coverage ignore patterns
-  coveragePathIgnorePatterns: ['index.ts'],
+  coveragePathIgnorePatterns: ['index.ts', '.d.ts'],
```

Comment:
Ignores the types for the mdx file 

![Screenshot 2024-11-15 at 12 33 04 PM](https://github.com/user-attachments/assets/943fac34-5be0-443b-b5b7-22bfbbb0c83b)